ethereum/core/src/main/java/org/hyperledger/besu/ethereum/mainnet/precompiles/privacy/FlexiblePrivacyPrecompiledContract.java

```diff
@@ -107,7 +107,7 @@ public PrecompileContractResult computePrecompile(
 
     final Hash pmtHash = messageFrame.getContextVariable(KEY_TRANSACTION_HASH);
 
-    final String key = input.slice(0, 32).toBase64String();
```

Comment:
what if the input is greater than 32 bytes?

Reply:
As a result of testing, An exception was occurs. I think, this exception is caused by that  the tessera node can’t find corresponding enclavekey.

```
“timestamp”:“2022-03-29T05:45:41,018"
“container”:“ip-172-31-57-54.ap-northeast-2.compute.internal”
"level”:“DEBUG”,“thread”:“EthScheduler-Workers-1"
“class”:“FlexiblePrivacyPrecompiledContract”
“message”:“Can not fetch private transaction payload with key kSUnNL+W53+lT7qc4POA380jiArDnj8i5ePSa89zLEurzQ==“
”throwable”:” org.hyperledger.besu.enclave.EnclaveClientException: Message with hash= was not found 
  at org.hyperledger.besu.enclave.Enclave.handleJsonResponse(Enclave.java:160)
  at org.hyperledger.besu.enclave.Enclave.lambda$receive$2(Enclave.java:85)
  ...
```

Reply:
It doesn't make sense for the enclaveKey to be null - you need to figure out why that's happening.

Reply:
the enclaveKey might be able to null by mistake. An asynchronous problem can be exemplified.